src/main/java/core/basesyntax/WorkWithFile.java

```diff
@@ -1,7 +1,67 @@
 package core.basesyntax;
 
+import java.io.BufferedReader;
+import java.io.FileReader;
+import java.io.IOException;
+import java.nio.file.Files;
+import java.nio.file.Path;
+
 public class WorkWithFile {
+    private static final String FILE_SEPARATOR = ",";
+    private static final String SUPPLY_OPERATION = "supply";
+    private static final String BUYING_OPERATION = "buy";
+    private static final int OPERATION_INDEX = 0;
+    private static final int VALUE_INDEX = 1;
+
     public void getStatistic(String fromFileName, String toFileName) {
+        int buy = 0;
+        int supply = 0;
+        try (BufferedReader reader = new BufferedReader(new FileReader(fromFileName))) {
+            String line = reader.readLine();
```

Comment:
move `line = reader.readline` inside the while ((line = reader.readLine()) != null) ' , so you don't need to duplicate it in line 29.

Reply:
Okay, I will try not to make this mistake in the future